app/src/userpages/components/ProductsPage/Members/index.jsx

```diff
@@ -7,6 +7,7 @@ import { withRouter } from 'react-router-dom'
 import cx from 'classnames'
 
 import CoreLayout from '$shared/components/Layout/Core'
+import CoreLayoutStyles from '$shared/components/Layout/core.pcss'
```

Comment:
```suggestion
import coreLayoutStyles from '$shared/components/Layout/core.pcss'
```

Reply:
That'd make it follow that little convention for pcss imports… I'll update! 👍